README.md

```diff
@@ -43,18 +43,16 @@ Alternatively, download
 $ yarn add rrule
 ```
 
-```javascript
-var RRule = require('rrule').RRule
-var RRuleSet = require('rrule').RRuleSet
-var rrulestr = require('rrule').rrulestr
+```es6
+import { RRule, RRuleSet, rrulestr } from 'rrule/es6'
```

Comment:
💛 I don't think its mentioned anywhere in the README but we should point out that you can use

```
import { RRule, RRuleSet, rrulestr } from 'rrule'
```

to pull in RRule that requires no transpiling